spring-boot/messaging-kafka/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id("org.springframework.boot")
+    id("io.spring.dependency-management")
+
+    kotlin("jvm")
+    kotlin("plugin.spring")
+}
+repositories {
+    mavenCentral()
+}
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("org.springframework.kafka:spring-kafka")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    implementation("org.jetbrains.kotlin:kotlin-reflect")
+
+    testImplementation("org.springframework.boot:spring-boot-starter-test")
+    testImplementation("org.springframework.kafka:spring-kafka-test")
+    testImplementation("io.mockk:mockk")
+    testImplementation("com.ninja-squad:springmockk")
+    testImplementation("org.testcontainers:testcontainers")
+    testImplementation("org.testcontainers:kafka:1.16.2")
```

Comment:
version should be managed in root build file

Reply:
done